weldx/__init__.py

```diff
@@ -9,7 +9,17 @@
 import weldx.transformations
 import weldx.utility
 import weldx.welding
+
+# class imports to weldx namespace
+from weldx.asdf.tags.weldx.core.iso_groove import get_groove
 from weldx.constants import WELDX_QUANTITY as Q_
+from weldx.core import MathematicalExpression, TimeSeries
+from weldx.geometry import ArcSegment, Geometry, LineSegment, Profile, Shape, Trace
```

Comment:
We should add the other classes of the package too (trace segments, variable profile)

Reply:
> We should add the other classes of the package too (trace segments, variable profile)

I skipped them since they are not actively used anywhere currently I think and I would prefer not cluttering the weldx namespace too much.
But feel free to add them

Reply:
> I skipped them since they are not actively used anywhere currently I think
At least the trace segments should appear in every file that has a geometry since you need them to create a trace. However, we can always add more later, so just merge it.